comp/README.md

```diff
@@ -88,6 +88,20 @@ Package runner implements a component to generate the 'resources' metadata paylo
 
 Package runner implements a component to generate metadata payload at the right interval.
 
+## [comp/otelcol](https://pkg.go.dev/github.com/DataDog/dd-agent-comp-experiments/comp/otelcol) (Component Bundle)
+
+*Datadog Team*: opentelemetry
+
+Package otelcol contains the OpenTelemetry Collector bundle pipeline to be included
```

Comment:
The fact that we use the Collector is an implementation detail. The feature is called OTLP ingest, I think we should keep naming it as such so long as we don't make changes to the product

```suggestion
Package otelcol contains the OTLP ingest bundle pipeline to be included
```